### PR TITLE
Added a set of atmospheric analysis sensors...

### PR DESCRIPTION
**Pull request by Dunkhan**
Added a set of atmospheric analysis sensors to the integrated electronics module

Consists of 7 new input parts. Pressure, temperature, o2 n2 co2 and phoron levels. The last part is a general atmospheric analyser which has all 6 of those functions in one unit. The multi-purpose part uses triple the complexity and power so it is only worthwhile if you need more than 2 of the output pins.

### CODE REVIEW

**Comment by Dunkhan** [comment]
Please note: There are no changes to the icons, all parts use an existing icon - the one from the advanced medical sensor. I did not feel that this was important to change but someone will probably want to at some point.

File: code/modules/integrated_electronics/subtypes/input.dm [hunk 1]
```diff
@@ -619,3 +619,235 @@
 	activate_pin(2)
 
 
+/obj/item/integrated_circuit/input/atmo_scanner
+	name = "integrated atmospheric analyser"
+	desc = "The same atmospheric analysis module that is integrate into every pda.  \
```
Comment (Belsima) [inline code/modules/integrated_electronics/subtypes/input.dm]: integrated

PDA

also im not sure thats true either

Comment (Dunkhan) [reply code/modules/integrated_electronics/subtypes/input.dm]: Could you elaborate? I am not sure what you mean.

Comment (Belsima) [reply code/modules/integrated_electronics/subtypes/input.dm]: PDAs only sometimes do; not all of them do. 'every' PDA is incorrect.

Comment (Neerti) [reply code/modules/integrated_electronics/subtypes/input.dm]: PDAs still have an atmos sensor function, and its the only portable one that auto-updates iirc.

**Pull request merged**